src/zarr/api/synchronous.py

```diff
@@ -895,6 +902,212 @@ def create_array(
                 storage_options=storage_options,
                 overwrite=overwrite,
                 config=config,
+                write_data=write_data,
+            )
+        )
+    )
+
+
+def from_array(
+    data: Array | npt.ArrayLike,
+    store: str | StoreLike,
+    write_data: bool = True,
+    *,
+    name: str | None = None,
```

Comment:
```suggestion
def from_array(
    store: str | StoreLike,
    *,
    data: Array | npt.ArrayLike,
    write_data: bool = True,
    name: str | None = None,
```

lets make everything except `store` keyword-only, to match `create_array`